src/sage/combinat/designs/orthogonal_arrays.py

```diff
@@ -1543,8 +1553,36 @@ def OA_relabel(OA,k,n,blocks=tuple(),matrix=None):
     if matrix:
         OA = [[matrix[i][j] if j is not None else None for i,j in enumerate(R)] for R in OA]
 
+    if symbol_list:
+        mapping = {index: symbol for index, symbol in enumerate(symbol_list)}
+        OA = [[mapping[element] for element in row] for row in OA]
     return OA
 
```

Comment:
There should be two blank lines between function definitions in the module namespace, see [PEP8, section blank lines](https://peps.python.org/pep-0008/#blank-lines).